docs/resources/core/dhcp_option.md

```diff
@@ -59,8 +59,10 @@ resource "oci_core_dhcp_options" "dhcp-options2" {
 
 The following arguments are supported:
 
-* `compartment_id` - (Required) The OCID of the compartment.
-* `vcn_id` - (Required) The OCID of the VCN.
+* `compartment_id` - (Required) The OCID of the compartment. Do not specify with `manage_default_resource_id`.
```

Comment:
isnt this optional then?

Reply:
You raise a good point. We need a way to say that you either must specify a compartment/vcn id or you must specify a default resource id. I will figure out a way to explain this.